projects/ng-components/lib/radio-group/radio-group.component.ts

```diff
@@ -104,8 +104,10 @@ export class RadioGroupComponent implements ControlValueAccessor, AfterViewCheck
         if (!this.didFocus && this.list && this.list.length) {
             const currentFocused: number = this.list.findIndex(e => e && this.selectedValue && e.value === this.selectedValue.value);
             if (currentFocused > -1 && this.radioRefs.toArray()[currentFocused] && this.radioRefs.toArray()[currentFocused].nativeElement) {
-                this.radioRefs.toArray()[currentFocused].nativeElement.focus();
-                this.didFocus = true;
+                setTimeout(() => {
+                    this.radioRefs.toArray()[currentFocused].nativeElement.focus();
```

Comment:
Can you just save guard this?
```suggestion
                    this.radioRefs.toArray()[currentFocused].nativeElement?.focus();
```